### PR TITLE
Synthetic dataset generation

### PR DESCRIPTION
**Pull request by martbern**

### CODE REVIEW

**Comment by martbern** [comment]
Generates an exampel dataframe for when we are generating evaluation plots. Then we can develop locally, and move to Overtaci when we want the real results.

File: src/data/generate_synthetic_dataset_for_eval.py [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+    msg = Printer(timestamp=True)
+    base = pd.Timestamp.today()
+    n_patients = 100_000
+
+    df = pd.DataFrame()
+
+    df["dw_ek_borger"] = [np.random.randint(0, 100_000) for n in range(n_patients)]
+
+    # Generate timestamps
+    df["timestamp"] = [base for n in range(n_patients)]
```
Comment (HLasse) [inline src/data/generate_synthetic_dataset_for_eval.py:71]: Hurtigere med det her?
```suggestion
    df["timestamp"] = [base] * n_patients
```

File: src/data/generate_synthetic_dataset_for_eval.py [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+        np.random.choice(series.index, int(len(series) * prop), replace=False)
+    ] = null_value
+
+    return series
+
+
+if __name__ == "__main__":
+    msg = Printer(timestamp=True)
+    base = pd.Timestamp.today()
+    n_patients = 100_000
```
Comment (HLasse) [inline src/data/generate_synthetic_dataset_for_eval.py:64]:
```suggestion
    n_rows = 100_000
```

**Review by HLasse — COMMENTED** [review]
Very minor, LGTM!

**Pull request merged**